deploy/helm/sumologic/values.yaml

```diff
@@ -177,7 +177,7 @@ sumologic:
           name: traces
           config-name: endpoint-traces
           properties:
-            content_type: Zipkin
+            content_type: OtlpHttpProto
```

Comment:
Is that a breaking change? We need to make sure it's not if that is to be included in v2.

Reply:
Hey @mat-rumian should we call that change out in the changelog as well?

Reply:
Hey, I've reverted the change of the content_type.